### PR TITLE
[libdwarf] fix hooks and conventions

### PR DESCRIPTION
**Pull request by mathbunnyru**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libdwarf/all/test_package/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,6 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
-set(CMAKE_VERBOSE_MAKEFILE TRUE)
```
Comment (mathbunnyru) [inline recipes/libdwarf/all/test_package/CMakeLists.txt:4]: bincrafters-conventions doesn't remove empty lines, will delete manually.

**Comment by mathbunnyru** [comment]
@ericLemanissier there is a room to improve `bincrafters-conventions` - I think it could easily add newlines.

**Comment by ericLemanissier** [comment]
This is more @Croydon's field than mine. can you report this on https://github.com/bincrafters/community/issues ?

**Comment by mathbunnyru** [comment]
Oh, sorry, mentioning you was a mistake :)

**Comment by mathbunnyru** [comment]
https://github.com/bincrafters/community/issues/1140

**Review by ericLemanissier — APPROVED** [review]

**Review by Croydon — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by theirix — APPROVED** [review]

**Pull request merged**